content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -1,59 +1,59 @@
 # Sharing Experiments
 
-There are two types of remotes that can store experiments. Git remotes are
-distributed copies of the Git repository, for example on GitHub or GitLab.
+If your team uses [Git remotes] on a Git server or hosting (e.g. GitHub, GitLab,
+etc.) to collaborate on projects, then you can also use them to save and share
+DVC Experiments remotely. You will also need DVC
+[remote storage](/doc/command-reference/remote) setup.
 
-[DVC remotes](/doc/command-reference/remote) on the other hand are
-storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
-configure with `dvc remote`. DVC uses them to store and fetch large files that
-don't normally fit inside Git repos.
+[git remotes]: https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
 
-DVC needs both kinds of remotes for backing up and sharing experiments.
+<details>
 
-Experiment files that are normally tracked in Git (like code versions) are
-shared using Git remotes, and files or directories tracked with DVC (like
-datasets) are shared using DVC remotes.
+## ⚙️ Expand to learn more.
 
-> See [Git remotes guide] and `dvc remote add` for information on setting them
-> up.
+Sharing experiments is similar to [sharing regular project data] and artifacts
+by synchronizing from remotes. DVC-tracked data, models, etc. are in your
+project's <abbr>cache</abbr> and thus will be transferred to/from
+[remote storage](/doc/command-reference/remote) (e.g. Amazon S3 or Google
+Drive). Small files like experimental code and
+[DVC metafiles](/doc/user-guide/project-structure) files are uploaded or
+downloaded with Git automatically as needed.
 
-[git remotes guide]:
-  https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes
+[sharing regular project data]: /doc/use-cases/sharing-data-and-model-files
 
-Normally, there should already be a Git remote called `origin` when you clone a
-repo. Use `git remote -v` to list your Git remotes:
+You can check you have all the necessary remotes setup with `git remote -v` and
+`dvc remote list`:
 
 ```dvc
 $ git remote -v
-origin  https://github.com/iterative/example-dvc-experiments (fetch)
-origin  https://github.com/iterative/example-dvc-experiments (push)
-```
-
-Similarly, you can see the DVC remotes in you project using `dvc remote list`:
+origin  git@github.com:iterative/get-started-experiments.git (fetch)
+origin  git@github.com:iterative/get-started-experiments.git (push)
 
-```dvc
 $ dvc remote list
-storage https://remote.dvc.org/example-dvc-experiments
+storage s3://mybucket/my-dvc-store
 ```
 
-## Uploading experiments to remotes
+</details>
+
+> ⚠️ Note that only [SSH Git URLs] are compatible with DVC Experiment sharing.
```

Comment:
But the note says "only SSH Git URLs", not "URLs in the form of Git SSH URLs." The former phrasing also means "no file URLs or paths, only SSH Git URLs."

Reply:
It says only SSH URLs **are compatible**. That does not imply that only URLs are accepted by `exp push`.

Reply:
And in fact if you go to https://dvc.org/doc/command-reference/exp/push it only mentions Git remote names or URLs for `git_url` so if that info is incomplete (also paths are accepted) it's a more general problem not really specific to this guide only and especially not to this PR I think. It wasn't mentioned in the original PR either ([current published doc](https://dvc.org/doc/user-guide/experiment-management/sharing-experiments)).

Reply:
Extracted to https://github.com/iterative/dvc.org/issues/2992

Reply:
@iesahin `git_remote` does not accept paths. Additionally, the paragraph above reads "If your team uses Git remotes on a Git server or hosting" so here the Git remote will contain a URL, not a path.

Do you have a suggestion on how to improve the whole intro or some other section to leave room for Git remotes that are paths? TBH it seems like an edge case and more related documenting Git than DVC; It should be clear to people who have that setup. And you do use that in the example already.

And again, the current doc also doesn't explain path-based Git remotes. On the other hand we definitely want to put this warning about SSH URLs so this is an improvement.

Reply:
`git_remote` is a Git remote name, not a URL or path. You can define a `git_remote` in the form: 

```
git remote add myremote ~/myrepobackups/myrepo.git 
```

and `myremote` will work with `dvc exp push`. 

When you say _only_, it reads like the above form doesn't work.

Reply:
yep, agreed with Emre on this from a brief look. Local path is also URI. Btw, what are other options besides SSH? 

HTTP is not supported (no writable), what else?

Reply:
may be it's easier to say that HTTP URLs are not supported?

Reply:
> Local path is also URI

@iesahin @shcheklein This is a larger issue then, since we published it also in https://dvc.org/doc/user-guide/troubleshooting#git-auth and don't mention the edge case of path-based Git remote definitions. Should we address it in this PR?

> say that HTTP URLs are not supported

HTTP URLs are supported, even for pushing, if the remote accepts unauthenticated writes. It's more about the authentication. Only SSH auth is supported. Local paths never require auth AFAIK.

Reply:
OK what about "Note that DVC can only authenticate with Git remotes via [SSH URLs]." ? Using that for now. WDYT @iesahin @shcheklein ?